ui/components/src/TimeSeriesTooltip/tooltip-model.ts

```diff
@@ -109,12 +110,9 @@ export const useMousePosition = (): CursorData['coords'] => {
           y: e.clientY,
         },
         plotCanvas: {
-          x: e.offsetX,
```

Comment:
See inline comment below, this is the most important part of the fix to use [zrender](https://github.com/ecomfe/zrender) mousemove coords